src/Microsoft.Diagnostics.Monitoring.WebApi/Utilities/StackUtilities.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.Diagnostics.Monitoring.WebApi
+{
+    internal static class StackUtilities
+    {
+        public static string GenerateStacksFilename(IEndpointInfo endpointInfo)
+        {
+            return FormattableString.Invariant($"{Utilities.GetFileNameTimeStampUtcNow()}_{endpointInfo.ProcessId}.stacks");
```

Comment:
I think this should be `.txt` for plain text and `.json` for json depending on the Accept content type header.